### PR TITLE
Fix #4669: tasks per node configureable 

### LINKED ISSUE #4669: Enable specifying number of cores per node on nersc
Allow the user to specify the number of cores per node (tasks per node) when running a simulation on sbatch.

### PR DESCRIPTION
**Pull request by gurhar1133**
Tasks per node is now a configurable field

### CODE REVIEW

File: sirepo/package_data/static/json/srw-schema.json [hunk 1]
```diff
@@ -834,11 +838,7 @@
         },
         "multiElectronAnimation": {
             "_super": ["_", "model", "_INTENSITY_REPORT"],
-            "jobRunMode": ["Execution Mode", "JobRunMode", "parallel"],
-            "sbatchCores": ["Cores", "Integer", 128],
-            "sbatchHours": ["Hours", "Float", 0.4],
-            "sbatchQueue": ["Queue", "NERSCQueue", "debug"],
-            "sbatchProject": ["Project", "OptionalString", ""],
+            "_super": ["_", "model", "sbatchFields"],
```
Comment (e-carlin) [inline sirepo/package_data/static/json/srw-schema.json:841]: I don't think having two _super keys works. At the very least the view is messed up (see screenshot). To replicate yourself start a multiElectronAnimation -> pencil icon -> view tab. There may be other problems.

Screen Shot 2022-08-16 at 2 03 22 PM

**Review by e-carlin — CHANGES_REQUESTED** [review]
I left some comments. Once those are cleaned up I'll give this a run at NERSC and report back.

File: sirepo/package_data/static/json/srw-schema.json (hunk 1 above)
Comment (gurhar1133) [reply sirepo/package_data/static/json/srw-schema.json:841]: got rid of the two supers

**Comment by gurhar1133** [comment]
made changes @e-carlin

**Comment by e-carlin** [comment]
Worked well at NERSC!

**Pull request merged**